src/main/views/includes/scripts.njk

```diff
@@ -7,6 +7,8 @@
 <script src="{{ asset_paths['js'] }}/detectJavascript.js"></script>
 <script src="{{ asset_paths['js'] }}/analytics.js"></script>
 <script src="{{ asset_paths['js'] }}/jquery-prevent-double-submit.js"></script>
```

Comment:
@hmcts/cmc 
We have one jquery implementation targeting page pay-by-account.njk - src/main/features/claim/views/pay-by-account.njk implemented a whilst back. It doesn't disable button but submit page once.

Reply:
Remove old jquery based solution then. Supporting both is not an option.